### PR TITLE
hoc2019: Add new robotics partner to hourofcode.com/

### PR DESCRIPTION
**Pull request by breville**
Adds a new robotics partner and updates the classroom count on https://hourofcode.com/.

### desktop

Screenshot 2019-10-24 12 27 58

### mobile

Screenshot 2019-10-24 12 27 47

### CODE REVIEW

**Comment by agealy** [comment]
I'm a little concerned with how big this ends up being on mobile. Is there a way we could scale down each partner box such that they all fit on the same line?

**Comment by breville** [comment]
@agealy Good point.  After we discussed in person, I'll merge this for now and we'll see if it feels egregious once we try it on a few devices.

**Review by maddiedierker — APPROVED** [review]

**Review by Erin007 — APPROVED** [review]

**Pull request merged**
